Sources/Prometheus/MetricTypes/Gauge.swift

```diff
@@ -84,25 +86,28 @@ public class Gauge<NumType: Numeric, Labels: MetricLabels>: Metric, PrometheusHa
     /// - Parameters:
     ///     - amount: Amount to increment the Gauge with
     ///     - labels: Labels to attach to the value
+    ///     - done: Completion handler
+    ///     - observedValue: Value written to the Gauge
     ///
-    public func inc(_ amount: NumType, _ labels: Labels? = nil, _ done: @escaping (NumType) -> Void = { _ in }) {
+    public func inc(_ amount: NumType, _ labels: Labels? = nil, _ done: @escaping (_ observedValue: NumType) -> Void = { _ in }) {
```

Comment:
I prefer not to name closure parameters.
```suggestion
    public func inc(_ amount: NumType, _ labels: Labels? = nil, _ done: @escaping (NumType) -> Void = { _ in }) {
```